src/frontend/src/app/shared/services/graph.service.ts

```diff
@@ -22,7 +26,11 @@ export class GraphService {
   makeGraph(nodeMap: Map<string, Node>, forwards: Forward[]): void {
     this.initializeGraph();
     this.addNodesToGraph(nodeMap);
-    this.connectAllNodes(forwards);
+    try {
```

Comment:
What was the reason for using a try catch?

Reply:
If there is a forward that points to a node that doesn't exist cytoscape will throw an error. Because we still want to generate a graph even if the forwards aren't configured right we catch the cytoscape error.